README-dtq.md

```diff
@@ -21,20 +21,15 @@ DSpace uses GitHub to track issues:
 - Language is not properly showed in the Item View, it is `ces, zxx` instead of `Czech, Nolinguistic content`. Issue: https://github.com/dataquest-dev/DSpace/issues/253
 
 ### Tables which are not migrated yet:
-- resurcepolicy - automatically filled, 
-- schema_version - automatically filled, 
-- subscription, 
-- tasklistitem, 
-- webapp - automatically filled, 
-- license_ressource_user_allowance, 
-- user_metadata, 
-- verification_token
+- subscription, - Planned in CLARIN-DSpace7.5.
 
 ### Tables which are missing in the DSpace7.*., but they are in the CLARIN-DSpace5.* (they are not planned to migrate)
 - userconnection, 
 - license_file_download_statistic, 
 - piwik_reposr, 
 - shibboleth_attribute_mapping
+- verification_token
+- webapp
```

Comment:
this is not true, it is in dspace7

Reply:
@Paurikova2  So I will remove it from this list, right?

Reply:
yes, somewhere in my mind is that webapp data was empty, that was the reason why we didn't migrate it, but I am not sure... or it was imported somehow...magic... but again, i am not sure... I think it would be better put it to upper section.